site/app/templates/officeHoursQueue/QueueHistory.twig

```diff
@@ -84,3 +92,37 @@
     </tbody>
   </table>
 </div>
+
+<script>
+
+    document.addEventListener('DOMContentLoaded', function() {
+      var sortIndicator = document.getElementById('sortIndicator');
+      if(Cookies.get('sortStateTimeEntered')=='up'){
+        //sortIndicator.innerText = "▲";
+        sortIndicator.innerHTML = '<i class="fas fa-sort-up"></i>'
+      }
+      if(Cookies.get('sortStateTimeEntered')=='off'){
+        sortIndicator.innerText = "";
+      }
+      if(Cookies.get('sortStateTimeEntered')=='down'){
+        sortIndicator.innerHTML = '<i class="fas fa-sort-down"></i>';
+      }
+    });
+
+    function toggleSort(column) {//per click actions handler
+        if (column == 'HelpStartedBy') {
+            if (Cookies.get('sortStateTimeEntered') == "off") {
+                Cookies.set('sortStateTimeEntered', 'up'); // Save to cookie
+                sortIndicator.innerHTML = '<i class="fas fa-sort-up"></i>'
+            } else if (Cookies.get('sortStateTimeEntered') == "up") {
+                Cookies.set('sortStateTimeEntered', 'down'); // Save to cookie
+                sortIndicator.innerHTML = '<i class="fas fa-sort-down"></i>';
+            } else if (Cookies.get('sortStateTimeEntered') == "down") {
+                Cookies.set('sortStateTimeEntered', 'off'); // Save to cookie
+                sortIndicator.innerText = ""; // Reset to down arrow for off state
+                console.log(Cookies.get('sortStateTimeEntered'))
+            }
+        }
+        window.location.reload();
```

Comment:
I didn't like how the page would reload and send me up to the top every time I clicked the sort button. I think avoiding refreshing the page and just moving the table entries around would feel better to users.